.travis.yml

```diff
@@ -2,7 +2,14 @@ language: node_js
 
 node_js:
   - node
```

Comment:
While you are at it. Add Node 12 and Node 14 as well.

Reply:
Is this because publish will happen multiple times if we specify multiple node versions here?